tools/sm_teledump.lua

```diff
@@ -73,12 +70,16 @@ local MEMTRACK = { -- {{{
     { 0x0A68, 0x2, 'Flash suit' },
     { 0x0A76, 0x2, 'Hyper beam' },
     { 0x0AF6, 0x2, 'Samus X' },
+    { 0x0AF8, 0x2, 'Samus subpixel X' },
     { 0x0AFA, 0x2, 'Samus Y' },
+    { 0x0AFC, 0x2, 'Samus subpixel Y' },
     { 0x0B3F, 0x2, 'Blue suit' },
-    { 0xD7C0, 0x60, 'SRAM copy' }, -- Prob not doing much?
-    { 0xD820, 0x100, 'Events, Items, Doors' },
-    -- { 0xD840, 0x40, 'Items' },
-    -- { 0xD8B0, 0x40, 'Doors' },
+    -- { 0xD7C0, 0x60, 'SRAM copy' },
+    -- { 0xD820, 0x100, 'Events, Items, Doors' },
+    { 0xD820, 0x04, 'Events' },
+    { 0xD828, 0x08, 'Bosses' },
+    { 0xD870, 0x14, 'Items' },
+    { 0xD8B0, 0x16, 'Doors' },
```

Comment:
Yep we collected 96 bytes of SRAM for no reason, and also 256 bytes of "Events, Items, Doors" when we only needed 54 bytes.